CRM/Upgrade/Incremental/php/FiveThirtyEight.php

```diff
@@ -55,6 +55,47 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
    * (change the x in the function name):
    */
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_38_alpha1($rev) {
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Enable Payflow Pro Payment Processor Extension', 'enablePayflowProExtension');
+  }
+
+  public static function enablePayflowProExtension(CRM_Queue_TaskContext $ctx) {
+    $payflowProPaymentProcessorType = CRM_Core_DAO::singleValueQuery("SELECT id FROM civicrm_payment_processor_type WHERE class_name = 'Payment_PayflowPro'");
+    if ($payflowProPaymentProcessorType) {
+      $payflowProPaymentProcessorCount = CRM_Core_DAO::singleValueQuery("SELECT count(id) FROM civicrm_payment_processor WHERE payment_processor_type_id = %1", [1 => [$eWAYPaymentProcessorType, 'Positive']]);
```

Comment:
@seamuslee001 looks like some copy & paste mishap at the end of this line ```$eWAYPaymentProcessorType```

Reply:
Fixed now